### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by Wangata**
Updated the ReadMe.md with some typos and some additions to it, I hope it was helpful. 
Kindly let me know if there's anything else I can do to improve the readMe. 
Thanks!

### CODE REVIEW

**Comment by jcubic** [comment]
Thanks for the PR, but I can't merge it in current form. If you want to modify the README you need to change README in templates directory and use n devel branch. I don't accept PR to master.

Also not every change is correct.

File: README.md [hunk 1]
```diff
@@ -22,7 +22,7 @@
 It is based on the Scheme dialect and the R5RS/R7RS specifications. It has extensions to make it easier
 to interact with JavaScript. It work both in the browser and with Node.js.
 
-The name is a recursive acronym which stands for LIPS Is Pretty Simple.
+The name is a recursive acronym which stands for LIPS it's pretty simple.
```
Comment (jcubic) [inline README.md:25]: This is an acronym it's not an error that it's written this way. LIPS Is Pretty Simple is a [recursive acronym](https://en.wikipedia.org/wiki/Recursive_acronym) where the first letter of the word is part of the name.

File: README.md [hunk 2]
```diff
@@ -80,15 +80,15 @@ https://github.com/jcubic/lips/blob/master/lib/js/bookmark.js
 Create any link in your bookmarks, edit it and copy paste the content of that file.
 Affter you click on the link it will create the REPL at the bottom of the page.
 (NOTE: It may not work on every page because of content security policy;
-e.g. google.com or gihub.com)
+e.g. google.com or github.com)
 
-If you have trouble with creating the bookmarklet you can open
-[LISP Scheme home page](https://lips.js.org/#bookmark) where you can
+If you have trouble with creating the bookmark you can open
```
Comment (jcubic) [inline README.md:85]: [bookmarklet](https://en.wikipedia.org/wiki/Bookmarklet) is the name of the bookmark that is also JavaScript code that you can execute.

**Review by jcubic — CHANGES_REQUESTED** [review]

**Comment by Wangata** [comment]
Thanks for the comments

File: README.md (hunk 1 above)
Comment (Wangata) [reply README.md:25]: Thanks I didn't know that, i appreciate the heads up i am going to read more on that.

**Comment by jcubic** [comment]
@Wangata so you don't want to contribute anymore?

**Comment by Wangata** [comment]
Hey bro, I do want to contribute hence I want to take time to take time and get an issue maybe you can recommend an issue

**Comment by jcubic** [comment]
This really conflict with the project so I need to close the issue. If you can find real grammar errors in the README you can fix them, but don't change the meaning of the text.

**Pull request closed without merging**